eng/pipelines/runtime.yml

```diff
@@ -383,6 +383,25 @@ jobs:
     jobParameters:
       liveRuntimeBuildConfig: release
 
+#
+# Build libraries using live CoreLib
+# These are part of the test coreclr generic test build.
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/libraries/build-job.yml
+    buildConfig: Release
+    platforms:
+    - OSX_x64
+    jobParameters:
+      liveRuntimeBuildConfig: release
+      condition: >-
+        and(
+          ne(variables['debugOnPrReleaseOnRolling'], 'Release'),
+          or(
+            eq(dependencies.checkout.outputs['SetPathVars_coreclr.containsChange'], true),
+            eq(variables['isFullMatrix'], true)))
```

Comment:
This was required because I chose to always build the generic coreclr tests on `OSX_x64`.  We could of course build them on another architecture.  We had suggested `OSX_x64`because it generally has the fastest build times.